internal/status/status.go

```diff
@@ -0,0 +1,133 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019-2021 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package status
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/onflow/flow-cli/internal/command"
+	"github.com/onflow/flow-cli/pkg/flowcli/services"
+	"github.com/onflow/flow-cli/pkg/flowcli/util"
+	"github.com/spf13/cobra"
+)
+
+type FlagsStatus struct {
+}
+
+var statusFlags = FlagsStatus{}
+
+
+type Result struct {
+	network  string
+	accessNode string
+	connectionError error
```

Comment:
```suggestion
	err error
```

I would just name this `err`, since the error returned isn't guaranteed to be a connection error. The Access API could return any other kind of error, too.

Reply:
It'd be good to print the error if it occurs, too

Reply:
Shall we just `fmt.Println` the error?